### PR TITLE
Generate protos on setup.py develop.

### PR DESCRIPTION
**Pull request by robertwb**
Works around the hack of having to run sdist as well as mentioned in https://docs.google.com/document/d/1bcl4RcYX-fWKsPSesnFkdq15oUKrLkAvQgOtsJwxEho/edit#

------------------------

Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Make sure there is a [JIRA issue](https://issues.apache.org/jira/projects/BEAM/issues/) filed for the change (usually before you start working on it).  Trivial changes like typos do not require a JIRA issue.  Your pull request should address just this issue, without pulling in other changes.
 - [ ] Format the pull request title like `[BEAM-XXX] Fixes bug in ApproximateQuantiles`, where you replace `BEAM-XXX` with the appropriate JIRA issue.
 - [ ] Write a pull request description that is detailed enough to understand:
   - [ ] What the pull request does
   - [ ] Why it does it
   - [ ] How it does it
   - [ ] Why this approach
 - [ ] Each commit in the pull request should have a meaningful subject line and body.
 - [ ] Run `mvn clean verify` to make sure basic checks pass. A more thorough check will be performed on your pull request automatically.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

### CODE REVIEW

File: sdks/python/setup.py [hunk 1]
```diff
@@ -132,13 +133,19 @@ def get_version():
...
     # pylint: disable=wrong-import-position
     import gen_protos
     class cmd(original_cmd, object):
       def run(self):
+        try:
+          # pylint: disable=wrong-import-position
+          import grpc_tools
+        except ImportError:
+          if require_grpc_tools:
+            ImportError("grpcio-tools required for development")
```
[the remark this answers is not in the document]
Comment (robertwb) [reply sdks/python/setup.py]: Yes.

[the remark this answers is not in the document]
Comment (robertwb) [reply sdks/python/setup.py]: This is a hard error here. If you're using setup.py develop, you should just install this.

[the remark this answers is not in the document]
Comment (udim) [reply sdks/python/setup.py]: I meant that you should add the `raise` keyword here.

[the remark this answers is not in the document]
Comment (udim) [reply sdks/python/setup.py]: friendly ping

[the remark this answers is not in the document]
Comment (lukecwik) [reply sdks/python/setup.py]: Robert is OOO till next week, he has been sporadically responding to things.

**Comment by robertwb** [comment]
The important part of this was done in https://github.com/apache/beam/pull/5146 .

**Pull request closed without merging**